exe_9.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+
+def mul_arrays(array1:np, array2:np)->np:

```

Comment:
Use more specific types for type hinting.
https://numpy.org/devdocs/reference/typing.html